src/util/objects.ts

```diff
@@ -27,3 +28,34 @@ export function isPlainObject(o: any): boolean {
   // Most likely a plain Object
   return true;
 }
+
+interface ElementDescription {
+  index: number;
+  count: number;
+}
+
+export function mostFrequent<T>(array: T[], type: AnySSZType): T[] {
+  const hashMap: Map<string, ElementDescription> = new Map<string, ElementDescription>();
+  array.forEach((e, index) => {
+    //We can optimize this by using faster hash like https://github.com/bevacqua/hash-sum
+    const hash = hashTreeRoot(e, type).toString('hex');
+    if(hashMap.has(hash)) {
+      const desc = hashMap.get(hash);
```

Comment:
I think it would be slightly better to do a `get` and check for null/undefined, instead of a `has` then `get`.